### DISCUSSION TITLE
Include meme in README

### DISCUSSION THREAD

**Discussion by ShikherVerma** [vercel/hyper#6137] [General]
**Is your feature request related to a problem? Please describe.**
People open meaningless issues 

**Describe the solution you'd like**
Include this meme in the README file
![aegpqBB_460swp](https://user-images.githubusercontent.com/10194699/143384808-db7d7bfd-590d-4b77-a205-99ebba0aebe6.jpg)

**Describe alternatives you've considered**
Disabling the issues feature on GitHub
Auto closing issues after 10 seconds of inactivity

**Additional context**
There are 700+ open issues
![Screenshot_20211125-105952-466](https://user-images.githubusercontent.com/10194699/143384972-6002417a-6425-4737-a3bf-919d0c10e1ab.png)

**Comment by claudecoded** [comment]
Man I need more memes like this, my readme is dead
